crd/multitenancy/api/v1alpha1/multitenantpodnetworkconfig.go

```diff
@@ -45,6 +48,9 @@ type MultitenantPodNetworkConfigSpec struct {
 	PodNetwork string `json:"podNetwork"`
 	// name of the requesting cx pod
 	PodName string `json:"podName,omitempty"`
+	// MACs of the IB devices to use for a pod
+	// +kubebuilder:validation:Optional
+	IBMACs []net.HardwareAddr `json:"IBMACs,omitempty"`
```

Comment:
I would expand the name to `IBMACAddresses`

Reply:
Done